ush/config_defaults.yaml

```diff
@@ -3017,19 +3026,19 @@ cpl_aqm_parm:
   DO_AQM_SAVE_AIRNOW_HIST: false
   DO_AQM_SAVE_FIRE: false
 
-  AQM_CONFIG_DIR: ""
-  DCOMINbio_dfv: ""
+  FIXaqmconfig: ""
+  FIXaqmbio: ""
   AQM_BIO_FILE: "BEIS_SARC401.ncf"
 
-  DCOMINdust_dfv: "/path/to/dust/dir"
+  FIXaqmdust: "/path/to/dust/dir"
   AQM_DUST_FILE_PREFIX: "FENGSHA_p8_10km_inputs"
   AQM_DUST_FILE_SUFFIX: ".nc"
 
-  DCOMINcanopy_dfv: "/path/to/canopy/dir"
+  FIXaqmcanopy: "/path/to/canopy/dir"
   AQM_CANOPY_FILE_PREFIX: "gfs.t12z.geo"
   AQM_CANOPY_FILE_SUFFIX: ".canopy_regrid.nc"
 
-  DCOMINfire_dfv: ""
+  DCOMINfire: ""
```

Comment:
What is the difference between `DCOMINfire` and `FIXaqmfire`? You've already defined `FIXaqmfire` above.

Reply:
DCOMINfire includes the real-time RAVE data files whereas FIXaqmfire keeps a dummy or climatology file for RAVE product (fire emissions)

Reply:
Got it. Please update the above description for them with your answer.